src/assets/styles/main.scss

```diff
@@ -0,0 +1,63 @@
+$body-size: 1rem;
+$primary: rgb(89, 81, 139);
+$link: hsl(194deg 50% 50%);
+$info: #87ADB3;
+$primary-color: $link;
+
+@import "../../../node_modules/bulma/bulma.sass";
+@import '../../../node_modules/vue-datepicker-next/scss/index.scss';
+
+
+.disabled {
+  background-color: #efefef !important;
+  color: $grey !important;
```

Comment:
Why the importants?

Reply:
probably shouldn't need to be any more - I'll look through all the importants and see what I can get rid of

Reply:
I was able to get rid of most of them, the couple that are left have comments now - thanks!